pelops/features/keras_model.py

```diff
@@ -3,20 +3,39 @@
 from keras.applications.resnet50 import preprocess_input
 from keras.models import load_model
```

Comment:
It might be worth combining the three imports from keras.models into a single line